ex1B.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+def remove_dividend_numbers(num_list: List[int], divisor: int) -> List[int]:
```

Comment:
there should be a spare line between imports and the code.
use pycharm shortcut to reformat code:
ctrl+shift+l

Reply:
ctrl+alt+shift+l ?

Reply:
ctrl + alt + l ?